src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -3701,20 +3701,22 @@ int32_t SurgeGUIEditor::controlModifierClicked(CControl* control, CButtonState b
             }
          } // end vt_float if statement
 
-#if TARGET_VST3
-         auto hostMenu = addVst3MenuForParams(contextMenu, ptag, eid );
-#endif
+         #if TARGET_VST3
```

Comment:
Compiler directives shouldn't be tabbed like this. While technically #if can be anywhere, idiomatically almost every code base in the world keeps them at column 0 since they are compiler directives not code flow directives and don't match the same balance as the code. Can you put it back the way it was where you changed it?